library/std/src/os/linux/fs.rs

```diff
@@ -1,5 +1,7 @@
 #![stable(feature = "metadata_ext", since = "1.1.0")]
 
+//! Linux-specific extensions to primitives in the `std::fs` module.
+
```

Comment:
I prefer moving doc upper the stable attr.

Reply:
Ah I see, that's better for consistency. I moved it up, as well as for `sys/unix/ext/net.rs`